packages/core/package.json

```diff
@@ -44,43 +44,45 @@
     "resize-observer-polyfill": "^1.5.1"
   },
   "devDependencies": {
-    "@ember/optional-features": "^1.1.0",
+    "@ember/optional-features": "^1.3.0",
+    "@glimmer/component": "^1.0.0",
```

Comment:
Glimmer packages as dependencies necessary?

Reply:
They are added as `devDependencies` during [upgrade](https://www.ember-cli-diff.org/?#/v3.14.0/v3.16.0), but I think these (will) need to be `dependencies` since it's an addon